src/images/icons.svg

```diff
@@ -0,0 +1,61 @@
+        <symbol id="icon-arrow" viewBox="0 0 32 32">
+            <path
+                d="M16.746 7.652v17.019h-1.811V7.652l-8.509 8.509-1.28-1.28L15.841 4.186l10.695 10.695-1.28 1.28-8.509-8.509z" />
+        </symbol>
+        <symbol id="icon-inverted-commas" viewBox="0 0 32 32">
+            <path
+                d="M8.201 17.469c1.546-.613 2.785-.452 3.719.481s1.21 2.421.831 4.463c-.292 1.517-.933 2.683-1.925 3.5-.963.787-2.085 1.181-3.369 1.181-1.75 0-2.96-.671-3.631-2.013-.671-1.371-.802-3.208-.394-5.512.233-1.254.613-2.523 1.137-3.806s1.225-2.596 2.1-3.938 2.013-2.508 3.413-3.5 2.946-1.633 4.637-1.925l-.437 4.638c-1.429.292-2.698 1.035-3.806 2.231-1.108 1.167-1.867 2.567-2.275 4.2zm13.169 0c1.546-.613 2.785-.452 3.719.481s1.21 2.421.831 4.463c-.292 1.517-.933 2.683-1.925 3.5-.963.787-2.085 1.181-3.369 1.181-1.75 0-2.96-.671-3.631-2.013-.671-1.371-.802-3.208-.394-5.512.233-1.254.613-2.523 1.137-3.806s1.225-2.596 2.1-3.938 2.013-2.508 3.413-3.5 2.946-1.633 4.637-1.925l-.437 4.638c-1.429.292-2.698 1.035-3.806 2.231-1.108 1.167-1.867 2.567-2.275 4.2z" />
+        </symbol>
+        <symbol id="icon-heart" viewBox="0 0 32 32">
+            <path stroke-linejoin="round" stroke-linecap="round" stroke-width="2.4"
+                d="M27.787 6.147a7.345 7.345 0 0 0-5.187-2.15 7.33 7.33 0 0 0-5.187 2.15L16 7.56l-1.413-1.413a7.333 7.333 0 0 0-10.374 0 7.333 7.333 0 0 0 0 10.374L16 28.308l11.787-11.787a7.345 7.345 0 0 0 2.15-5.187 7.33 7.33 0 0 0-2.15-5.187z" />
+        </symbol>
+        <symbol id="icon-trash" viewBox="0 0 32 32">
+            <path stroke-linejoin="round" stroke-linecap="round" stroke-width="2.4"
+                d="M21.333 8V6.933c0-1.493 0-2.24-.291-2.811a2.666 2.666 0 0 0-1.165-1.165c-.57-.291-1.317-.291-2.811-.291h-2.133c-1.493 0-2.24 0-2.811.291-.502.256-.91.664-1.165 1.165-.291.57-.291 1.317-.291 2.811V8m2.667 7.333V22m5.334-6.667V22M4 8h24m-2.667 0v14.933c0 2.24 0 3.36-.436 4.216a3.996 3.996 0 0 1-1.748 1.748c-.856.436-1.976.436-4.216.436h-5.867c-2.24 0-3.36 0-4.216-.436a3.996 3.996 0 0 1-1.748-1.748c-.436-.856-.436-1.976-.436-4.216V8" />
+        </symbol>
+        <symbol id="icon-search" viewBox="0 0 32 32">
+            <path stroke-linejoin="round" stroke-linecap="round" stroke-width="2.667"
+                d="M14.667 25.333c5.891 0 10.667-4.776 10.667-10.667S20.558 3.999 14.667 3.999 4 8.775 4 14.666s4.776 10.667 10.667 10.667zM28 28l-5.8-5.8" />
+        </symbol>
+        <symbol id="icon-star" viewBox="0 0 32 32">
+            <path
+                d="M13.826 3.262c.684-2.106 3.663-2.106 4.348 0l1.932 5.945a2.285 2.285 0 0 0 2.174 1.579h6.251c2.214 0 3.135 2.833 1.344 4.135l-5.057 3.674a2.286 2.286 0 0 0-.83 2.556l1.931 5.945c.684 2.106-1.726 3.857-3.517 2.555l-5.057-3.674a2.286 2.286 0 0 0-2.687 0l-5.057 3.674c-1.791 1.302-4.202-.45-3.517-2.555l1.932-5.945a2.287 2.287 0 0 0-.83-2.556l-5.057-3.674c-1.791-1.302-.871-4.135 1.344-4.135h6.251c.99 0 1.868-.638 2.174-1.579l1.932-5.945z" />
+        </symbol>
```

Comment:
Please add 1 new "icon-logo" for header logo.
`<symbol id="icon-logo" viewBox="0 0 176 32">
      <path d="M6.64 17.707l-6.64-11.040h3.973l4.453 7.653h0.053l4.453-7.653h3.973l-6.64 11.040v7.627h-3.627v-7.627zM16.893 22.267v-12.533l3.067-3.067h9.6l3.067 3.067v12.533l-3.067 3.067h-9.6l-3.067-3.067zM27.64 22.24l1.36-1.36v-9.76l-1.36-1.36h-5.76l-1.36 1.36v9.76l1.36 1.36h5.76zM35.165 22.133v-15.467h3.627v14.213l1.36 1.36h5.227l1.36-1.36v-14.213h3.627v15.467l-3.2 3.2h-8.8l-3.2-3.2zM67.837 19.147v6.187h-3.627v-5.36l-1.6-1.787h-5.813v7.147h-3.627v-18.667h11.707l2.853 2.88v5.653l-1.813 1.84 1.92 2.107zM56.797 15.2h6.427l0.933-0.933v-3.627l-0.933-0.933h-6.427v5.493zM77.097 6.667h13.493v3.067h-9.867v4.72h9.093v3.013h-9.093v4.8h9.867v3.067h-13.493v-18.667zM92.603 6.667h3.253l8.32 12.667h0.053v-12.667h3.493v18.667h-3.253l-8.32-12.64h-0.053v12.64h-3.493v-18.667zM110.531 6.667h13.493v3.067h-9.867v4.72h9.093v3.013h-9.093v4.8h9.867v3.067h-13.493v-18.667zM140.703 19.147v6.187h-3.627v-5.36l-1.6-1.787h-5.813v7.147h-3.627v-18.667h11.707l2.853 2.88v5.653l-1.813 1.84 1.92 2.107zM129.663 15.2h6.427l0.933-0.933v-3.627l-0.933-0.933h-6.427v5.493zM143.097 22.267v-12.533l3.067-3.067h9.28l3.013 3.013v2.853h-3.627v-1.52l-1.253-1.253h-5.493l-1.36 1.36v9.76l1.36 1.36h5.547l1.307-1.307v-2.907h-4v-3.093h7.52v7.333l-3.067 3.067h-9.227l-3.067-3.067zM165.104 17.707l-6.64-11.040h3.973l4.453 7.653h0.053l4.453-7.653h3.973l-6.64 11.040v7.627h-3.627v-7.627z"></path>
    </symbol>`

Reply:
OK! Done 👌